boot/bootdata/hivesys.inf

```diff
@@ -444,7 +444,7 @@ HKLM,"SYSTEM\CurrentControlSet\Control\CoDeviceInstallers","{4D36E97D-E325-11CE-
 
 ; Default computer name settings
 HKLM,"SYSTEM\CurrentControlSet\Control\ComputerName",,0x00000012
-HKLM,"SYSTEM\CurrentControlSet\Control\ComputerName\ComputerName","ComputerName",0x00000002,"COMPUTERNAME"
+HKLM,"SYSTEM\CurrentControlSet\Control\ComputerName\ComputerName","ComputerName",0x00000002,"ROSHOST"
```

Comment:
Why is this change an "improvement" ?

Reply:
`COMPUTERNAME` doesn't describe the computer's type.

Reply:
but this has nothing to do with the computer's type. computer name is just something to distinguish it on the (internal) network where it is connected.

Reply:
I am not a fan of PR5853. It simplifies for potential remote attackers to identify the OS the host is running. Therefore simplifying their remote attacks. And imho it doesn't offer anything helpful in return. Also it makes us drift away from Windows further. I don't see the motivation or real gain of that change.
If we want to harmonize to something, it should ideally match the Windows behavior, or should be at least not expose the OS the host is running. I do perceive `COMPUTERNAME` as a better choice than `ROSHOST'.
I do also like the idea of the random-part-addendum to avoid collisions if two or more ros hosts would be booted in the same subnet without taking care of the name by the user during installation (just repeatedly hammering ENTER).